src/main/java/bio/terra/cli/command/workspace/Create.java

```diff
@@ -34,6 +34,7 @@ protected void execute() {
     Workspace workspace =
         Workspace.create(
             id,
+            CloudPlatform.GCP, // moved from terra/cli/service/WorkspaceManagerService.java
```

Comment:
Delete comment, I don't think it's necessary. Though this would be a useful comment:

// Currently only GCP is supported.
CloudPlatform.GCP,

Reply:
Done